client/client-multi/client-c/src/kaa/utilities/kaa_aes_rsa.c

```diff
@@ -106,3 +128,44 @@ int rsa_sign(mbedtls_pk_context *pk, const uint8_t *input,
 
     return ret ? KAA_ERR_BADDATA : KAA_ERR_NONE;
 }
+
+kaa_error_t rsa_encrypt(const uint8_t *key, size_t key_size, const uint8_t *input,
+        size_t input_len, uint8_t *output)
+{
+    if (!key || !key_size) {
+        return KAA_ERR_BADPARAM;
+    }
+
+    int ret = 0;
```

Comment:
`ret` value is not used. Move declaration down the line and make it declaration-initialization